diff --git a/src/s3fs.cpp b/src/s3fs.cpp
index e66977f..462fa82 100644
--- a/src/s3fs.cpp
+++ b/src/s3fs.cpp
@@ -92,9 +92,11 @@ bool pathrequeststyle             = false;
 bool is_specified_endpoint        = false;
 std::string program_name;
 std::string service_path          = "/";
-std::string host                  = "http://s3.amazonaws.com";
+//std::string host                  = "http://s3.amazonaws.com";
+std::string host                  = "http://s3.cn-north-1.amazonaws.com.cn";
 std::string bucket                = "";
-std::string endpoint              = "us-east-1";
+//std::string endpoint              = "us-east-1";
+std::string endpoint              = "cn-north-1";
